chain/chain/src/store_validator/validate.rs

```diff
@@ -115,13 +116,21 @@ pub(crate) fn head_tail_validity(sv: &mut StoreValidator) -> Result<(), StoreVal
         sv.store.get_ser::<BlockHeight>(ColBlockMisc, CHUNK_TAIL_KEY),
         "Can't get Chunk Tail from storage"
     );
+    let fork_tail_db = unwrap_or_err!(
+        sv.store.get_ser::<BlockHeight>(ColBlockMisc, FORK_TAIL_KEY),
+        "Can't get Chunk Tail from storage"
+    );
     if tail_db.is_none() && chunk_tail_db.is_some() || tail_db.is_some() && chunk_tail_db.is_none()
     {
         err!("Tail is {:?} and Chunk Tail is {:?}", tail_db, chunk_tail_db);
     }
+    if tail_db.is_some() && fork_tail_db.is_none() {
+        err!("Tail is {:?} but fork tail is None", tail_db);
+    }
     if tail_db.is_some() && chunk_tail_db.is_some() {
```

Comment:
```suggestion
    if tail_db.is_some() {
```
We know that all other tails are set, right?